diff --git a/Artisan/Artisan/CraftingList/CraftingListUI.cs b/Artisan/Artisan/CraftingList/CraftingListUI.cs
index ad4e1ad..37b6e31 100644
--- a/Artisan/Artisan/CraftingList/CraftingListUI.cs
+++ b/Artisan/Artisan/CraftingList/CraftingListUI.cs
@@ -48,5 +48,7 @@ namespace Artisan.CraftingLists
         private static readonly ListFolders ListsUI = new();
 
-        private static bool GatherBuddy => DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var gb, false, true);
+        private static bool GatherBuddy =>
+            DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var gb, false, true) ||
+            DalamudReflector.TryGetDalamudPlugin("GatherBuddyReborn", out gb, false, true);
         private static bool ItemVendor => DalamudReflector.TryGetDalamudPlugin("Item Vendor Location", out var ivl, false, true);
 
diff --git a/Artisan/Artisan/UI/ListEditor.cs b/Artisan/Artisan/UI/ListEditor.cs
index 99eaf17..34360a3 100644
--- a/Artisan/Artisan/UI/ListEditor.cs
+++ b/Artisan/Artisan/UI/ListEditor.cs
@@ -168,5 +168,6 @@ internal class ListEditor : Window, IDisposable
 
     private static bool GatherBuddy =>
-        DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var gb, false, true);
+        DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var gb, false, true) ||
+        DalamudReflector.TryGetDalamudPlugin("GatherBuddyReborn", out gb, false, true);
 
     private static bool ItemVendor =>
diff --git a/Artisan/Artisan/UI/Tables/IngredientTable.cs b/Artisan/Artisan/UI/Tables/IngredientTable.cs
index ce70485..b122252 100644
--- a/Artisan/Artisan/UI/Tables/IngredientTable.cs
+++ b/Artisan/Artisan/UI/Tables/IngredientTable.cs
@@ -55,5 +55,6 @@ namespace Artisan.UI.Tables
 
         private static bool GatherBuddy =>
-            DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var _, false, true);
+            DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var _, false, true) ||
+            DalamudReflector.TryGetDalamudPlugin("GatherBuddyReborn", out _, false, true);
 
         private static bool ItemVendor =>
