diff --git a/Artisan/Artisan/CraftingList/CraftingListUI.cs b/Artisan/Artisan/CraftingList/CraftingListUI.cs
index 6e090f1..c0c7f02 100644
--- a/Artisan/Artisan/CraftingList/CraftingListUI.cs
+++ b/Artisan/Artisan/CraftingList/CraftingListUI.cs
@@ -47,7 +47,9 @@ internal class CraftingListUI
         public static int CurrentProcessedItemListCount;
         private static readonly ListFolders ListsUI = new();
 
-        private static bool GatherBuddy => DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var gb, false, true);
+        private static bool GatherBuddy =>
+            DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var gb, false, true) ||
+            DalamudReflector.TryGetDalamudPlugin("GatherBuddyReborn", out gb, false, true);
         private static bool ItemVendor => DalamudReflector.TryGetDalamudPlugin("Item Vendor Location", out var ivl, false, true);
 
         private static bool MonsterLookup => DalamudReflector.TryGetDalamudPlugin("Monster Loot Hunter", out var mlh, false, true);
diff --git a/Artisan/Artisan/UI/ListEditor.cs b/Artisan/Artisan/UI/ListEditor.cs
index a9c8a0e..ec42f1e 100644
--- a/Artisan/Artisan/UI/ListEditor.cs
+++ b/Artisan/Artisan/UI/ListEditor.cs
@@ -167,7 +167,8 @@ public override void PostDraw()
     }
 
     private static bool GatherBuddy =>
-        DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var gb, false, true);
+        DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var gb, false, true) ||
+        DalamudReflector.TryGetDalamudPlugin("GatherBuddyReborn", out gb, false, true);
 
     private static bool ItemVendor =>
         DalamudReflector.TryGetDalamudPlugin("Item Vendor Location", out var ivl, false, true);
diff --git a/Artisan/Artisan/UI/Tables/IngredientTable.cs b/Artisan/Artisan/UI/Tables/IngredientTable.cs
index 7a6ee18..a2676a1 100644
--- a/Artisan/Artisan/UI/Tables/IngredientTable.cs
+++ b/Artisan/Artisan/UI/Tables/IngredientTable.cs
@@ -54,7 +54,8 @@ internal class IngredientTable : Table<Ingredient>, IDisposable
         public readonly NumberForSaleColumn _numberForSaleColumn = new() { Label = "出售数量（所有服务器）" };
 
         private static bool GatherBuddy =>
-            DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var _, false, true);
+            DalamudReflector.TryGetDalamudPlugin("GatherBuddy", out var _, false, true) ||
+            DalamudReflector.TryGetDalamudPlugin("GatherBuddyReborn", out _, false, true);
 
         private static bool ItemVendor =>
             DalamudReflector.TryGetDalamudPlugin("ItemVendorLocation", out var _, false, true);
